diff -Naurp melange-2012.1~e3+git20120112.orig/etc/melange/melange.conf.sample melange-2012.1~e3+git20120112/etc/melange/melange.conf.sample
--- melange-2012.1~e3+git20120112.orig/etc/melange/melange.conf.sample	2012-01-06 06:55:40.000000000 -0500
+++ melange-2012.1~e3+git20120112/etc/melange/melange.conf.sample	2012-01-12 04:38:10.435639518 -0500
@@ -14,7 +14,7 @@ bind_port = 9898
 # SQLAlchemy connection string for the reference implementation
 # registry server. Any valid SQLAlchemy connection string is fine.
 # See: http://www.sqlalchemy.org/docs/05/reference/sqlalchemy/connections.html#sqlalchemy.create_engine
-sql_connection = sqlite:///melange_test.sqlite
+sql_connection = sqlite:///var/lib/melange/melange.sqlite
 # sql_connection = mysql://root:root@localhost/melange
 #sql_connection = postgresql://melange:melange@localhost/melange
 
